Assets/Scripts/Waypoint Scripts/WaypointProperties.cs

```diff
@@ -37,7 +37,10 @@ public class WaypointProperties : MonoBehaviour
         private LineRenderer LineProperties;
         private CapsuleCollider lineCollider;
 
-        private GameObject world;
+        private GameObject World;
+        private Vector3 WorldScaleInitial;
```

Comment:
What is the different between initial and current?

Reply:
@apollovision

Reply:
Initial is the Original position, which we need to have an idea of where we started. Current is where we currently are at. We need both to be know by how much to scale gameobjects. Current - Original = Scale.